src/components/result/FilterSelect/types.ts

```diff
@@ -11,10 +12,7 @@ export type FilterSelectProps = {
     name: string
     selected: boolean
   }[]
-  tradePeriodItems: {
-    code: string
-    name: string
-  }[]
+  tradePeriodItems: typeof TRADE_TYPES
```

Comment:
```suggestion
import type { TradeTypes } from '@types'

tradePeriodItems: TradeTypes
```
`TradeTypes` 타입 가져와서 사용해도 될 것 같습니다!

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/174/commits/07442cc6093ada90b9e21e15938cc48e34573066#diff-4753ae1ab8636d729c9f6593563402bb609e1919a773ca36d0a37196b5a3235b)